METRICS.md

```diff
@@ -0,0 +1,26 @@
+# Data we are collecting
```

Comment:
Maybe also link to the data review bug?

Reply:
There won't be a separate bug. chutten told me to just add it to the client-side bug since that makes it easier to see what it's related to. But I can link that bug / the corresponding section once we submit the data review request.